scripts/pi-hole/php/func.php

```diff
@@ -483,4 +483,15 @@ function getQueryTypeStr($querytype)
         return "TYPE".($qtype - 100);
 }
 
+// Returns the blocking status of pihole
+function piholeStatus()
+{
+    $setupVars = parse_ini_file("/etc/pihole/setupVars.conf");
+    if ($setupVars["BLOCKING_ENABLED"]=="1") {
```

Comment:
We do use `true/false` literally here
https://github.com/pi-hole/pi-hole/blob/363e2f10bb4f3717055a84c42954ca2b3bde1744/pihole#L225-L240

Reply:
I used this code in a new function.

Now, we have 2 functions:
- piholeStatusAPI(): returns "enabled" or "disabled";
- piholeStatus(): returns an integer representing the status.

PS: You can use this new `piholeStatus()` function on the status panel code.